services/masterdata/src/lib/components/logger-messages/logger-messages.component.ts

```diff
@@ -9,5 +9,6 @@ import { LoggerMessagesService } from "../../core/services";
 export class LoggerMessagesComponent implements OnInit {
 	constructor(public readonly logger: LoggerMessagesService) {}
 
+	// eslint-disable-next-line @angular-eslint/no-empty-lifecycle-method, @typescript-eslint/no-empty-function
 	ngOnInit(): void {}
```

Comment:
Just delete the `implements OnInit` and empty `ngOnInit` instead of disabling the rules.